### PR TITLE
Count executions by installation method

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,17 +11,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160913011553) do
+ActiveRecord::Schema.define(version: 20161206210706) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+  enable_extension "pg_stat_statements"
```
Comment (ohayon) [inline db/schema.rb]: Not totally sure how these ended up here? Going to look more into this.

Comment (KrauseFx) [reply db/schema.rb]: Might be related to your Postgres version? It's probably save to just delete them

File: db/schema.rb [hunk 2]
```diff
@@ -11,17 +11,25 @@
...
     t.date     "launch_date"
     t.integer  "launches"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
     t.integer  "ci"
     t.integer  "duration"
     t.integer  "duration_ci"
+    t.integer  "install_method_standalone", default: 0
```
Comment (KrauseFx) [inline db/schema.rb]: You have to commit the migration file, not just the updated schema 👍

**Review by KrauseFx — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 2 above)
Comment (ohayon) [reply db/schema.rb]: ohhhh. hmm. I didn't realize they didn't get committed there... I had to do a couple of migrations to get it right. Do you know how I should do that?

File: db/migrate/20161207145632_add_installation_method_to_bacons.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class AddInstallationMethodToBacons < ActiveRecord::Migration
+  def change
+    add_column :bacons, :install_method_rubygems, :integer, default: 0
+    add_column :bacons, :install_method_bundler, :integer, default: 0
+    add_column :bacons, :install_method_mac_app, :integer, default: 0
+    add_column :bacons, :install_method_standalone, :integer, default: 0
```
Comment (KrauseFx) [inline db/migrate/20161207145632_add_installation_method_to_bacons.rb:6]: Beautiful pre-fix and default values 👍

**Review by KrauseFx — APPROVED** [review]

**Pull request merged**